_posts/2017-05-25-0.1-auth.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Istio Auth: secure by default service to service communications"
+overview: Istio Auth 0.1 announcement
+published: true
+permalink: blog/istio-auth-for-microservices.html
+attribution: The Istio Team
+layout: post
+type: markdown
+redirect_from: "/blog/0.1-auth.html"
+---
+
+Conventional network security approaches fail to address security threats to distributed applications deployed in dynamic production environments. Today, we describe how Istio Auth enables enterprises to transform their security posture from just protecting the edge to consistently securing all inter-service communications deep within their applications. With Istio Auth, developers and operators can protect services with sensitive data against unauthorized insider access and they can achieve this without any changes to the application code!
+ 
+Istio Auth is the security component of the broader [Istio platform](https://istio.io/). It incorporates the learnings of securing millions of services in Google’s production environment.
+
+<!--end_excerpt-->
+
+## Background
+
+Modern application architectures are increasingly based on shared services that are deployed and scaled dynamically on cloud platforms. Traditional network edge security (e.g. firewall) is too coarse-grained and allows access from unintended clients. An example of a security risk is stolen authentication tokens that can be replayed from another client. This is a major risk for companies with sensitive data that are concerned about insider threats. Other network security approaches like IP whitelists have to be statically defined, are hard to manage at scale, and are unsuitable for dynamic production environments.
+
+Thus, security administrators need a tool that enables them to consistently, and by default, secure all communication between services across diverse production environments. 
+
+## Solution: Strong service identity and authentication
+
+Google has, over the years, developed architecture and technology to uniformly secure millions of services in its production environment against external attacks and insider threats. Key security principles include trusting the endpoints and not the network, strong mutual authentication based on service identity and service level authorization. Istio Auth is based on the same principles. 
+
+The alpha release of Istio Auth runs on Kubernetes and provides the following features: 
```

Comment:
We've systematically avoided using Alpha in any of our material, this is the 0.1 release....